roles/content_view_version_cleanup/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+content_view_version_cleanup_keep: 5
```

Comment:
Given this is a destructive operation, I lean towards having this be null by default and forcing the user to specify the value when using this role.

Reply:
+1

Reply:
you mean `null`, not `0`, right? as `0` would imply "don't keep things"

Reply:
Yes, `null`